cmd/convert.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/containers/podman-bootc/pkg/bib"
+	"github.com/containers/podman-bootc/pkg/user"
+	"github.com/containers/podman-bootc/pkg/utils"
+
+	"github.com/containers/podman/v5/pkg/bindings"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const bibImage = "quay.io/centos-bootc/bootc-image-builder:latest"
```

Comment:
Please make use of https://github.com/CentOS/centos-bootc/pull/453/commits/44561ffb9b77c3023e062ee2c44656b752c473ef instead